static/client.html

```diff
@@ -8,9 +8,15 @@
 %X_UA_COMPATIBLE%
   <title>Karma</title>
   <link href="favicon.ico" rel="icon" type="image/x-icon">
+  <meta name="color-scheme" content="dark light">
   <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no" />
   <style type="text/css">
+
+    :root {
+      color-scheme: light dark;
+    }
+
     iframe {
```

Comment:
I tried this with `./bin/karma start test/client/karma.conf.js` in the repository and the context is still light. Do you think we can make blank context also adjust when dark theme is active?

<img width="1552" alt="image" src="https://user-images.githubusercontent.com/823594/104418117-dfe5dd80-558f-11eb-8b9d-33fa33929a6d.png">